addons/survey/models/survey_user_input.py

```diff
@@ -401,14 +401,26 @@ def _prepare_statistics(self):
         scored_questions = self.mapped('predefined_question_ids').filtered(lambda question: question.is_scored_question)
```

Comment:
Also, maybe to ask to FRGI / LNA, when using 'review your answers' we get for instance this : https://tinyurl.com/2q9p5ey2 where you can see the background is green on the third choice (because score is greater than 0, but the answer is not correct.) Is this desired? (could be a task / fix in master or before in separate task) (other ongoing tasks may already change that)